src/components/App/App.js

```diff
@@ -132,7 +132,8 @@ App.propTypes = {
 	storeList: PropTypes.objectOf(PropTypes.object).isRequired,
 	users: PropTypes.arrayOf(PropTypes.object).isRequired,
 	slackUserFetchError: PropTypes.bool.isRequired,
-	currentUser: PropTypes.string
+	currentUser: PropTypes.string,
```

Comment:
Should this be `isRequired` since it's always passed? (Default value being set to `''` by reducer)